### DISCUSSION TITLE
Is there a way to filter issues by label using *not* like `not state::blocked`?

### DISCUSSION THREAD

**Discussion by zoechi** [magit/forge#707] [Ideas]
I only found a way to filter for issues that match a list of labels

**Comment by tarsius** [comment]
Not currently. I'll probably look into that when doing another round of improvements to the filtering, but that that is still a way of as I am focused on other packages (and when I return to this package, this probably won't be the first focus either).

**Reply by zoechi** [reply]
Thanks a lot! Then I can stop searching for now ;-)

**Reply by tarsius** [reply]
I've decided to not implement support for this.
